buildspecs/j9.flags

```diff
@@ -334,6 +334,17 @@ Currently only available on linux_x86 and win_x96 32 bit builds.</description>
 			<require flag="gc_realtime"/>
 		</requires>
 	</flag>
+	<flag id="gc_enableDoubleMap">
+		<description>Set on double map. Allows LINUX systems to double map arrays that are stored as arraylets.
+When enabled, a contiguous block of memory is created for each array which data surpasses the size of a region. This contiguous block represents the array as
+if the data was stored in a contiguous region of memory. All of the array data will be stored at their own region (not with spine); hence, all arraylets
+become discontiguous whenever this flag is enabled. Since there won’t be any empty arraylet leaves, then arrayoid NULL pointers are no longer required since
```

Comment:
Are there any footprint affects from allocating another page of data rather than reusing the spine?  What has been measured to validate this?

Reply:
Yes, in theory, it is expected to have negative side effects. Either footprint increase or more frequent GCs (if heap is fully expanded).

In real life, arraylet's are rare. Say, for 10 arrraylets live at a given moment and in average 1/2 of arraylet leaf lost, and 1000-2000 regions, we are talking 0.25-0.5% footprint increase.

The optimization is still disabled and will be for some time. Meanwhile, we will conduct more perf and (more importantly to me) endurance testing.

In a long run, we plan to create a hybrid arraylet/small object regions, where unused part of arraylet region is used for allocation of small objects. 

Based on result of the extra testing, estimates of what amount of work is required  for arraylet/small  and relative priorities to other work we'll decide if/when will enable it.